helios-services/src/main/java/com/spotify/helios/master/MasterService.java

```diff
@@ -186,6 +187,12 @@ public MasterService(final MasterConfig config,
 
     environment.lifecycle().manage(zooKeeperHealthChecker);
     environment.healthChecks().register("zookeeper", zooKeeperHealthChecker);
+
+    // Report health checks as a boolean gauge metric
```

Comment:
Ah thanks. For context, I changed it to an Integer because every other gauge is numeric, and I didn't know how to make jackson parse something that could be more than one type.